youtube_dl/extractor/gdcvault.py

```diff
@@ -133,16 +146,18 @@ def _real_extract(self, url):
             r's1\.addVariable\("file",\s*encodeURIComponent\("(/[^"]+)"\)\);',
             start_page, 'url', default=None)
         if direct_url:
-            video_url = 'http://www.gdcvault.com/' + direct_url
             title = self._html_search_regex(
                 r'<td><strong>Session Name</strong></td>\s*<td>(.*?)</td>',
                 start_page, 'title')
+            video_url = 'http://www.gdcvault.com' + direct_url
+            request = compat_urllib_request.Request(video_url, method='HEAD')
+            head = compat_urllib_request.urlopen(request)
 
             return {
                 'id': video_id,
                 'display_id': display_id,
                 'url': video_url,
-                'ext': 'flv',
+                'ext': determine_ext(head.geturl()),
```

Comment:
to make it work for both audios and videos.
for example this is for a video page:
```
youtube-dl -F http://gdcvault.com/play/1016925/Bejeweled-Blitz-one-year-in
[GDCVault] Bejeweled-Blitz-one-year-in: Downloading webpage
[info] Available formats for 1016925:
format code  extension  resolution note
0            flv        unknown
```
and this for an audio only page:
```
youtube-dl -F http://gdcvault.com/play/1019442/5-Key-Best-Practices-in
[GDCVault] 5-Key-Best-Practices-in: Downloading webpage
[info] Available formats for 1019442:
format code  extension  resolution note
0            mp3        unknown
```